second-edition/src/ch03-00-common-programming-concepts.md

```diff
@@ -1,20 +1,20 @@
-# Common Programming Concepts
+# Conceptos comunes de programación
```

Comment:
Hey dbucci24, do you mind if we use from now on capital letters for titles?
Like in this one will be 

Conceptos Comunes de Programación

So we resemble the english version

Thanks!

Reply:
Sorry, I missed it. I'll do it from the next one! 

Fixed.